app/assets/stylesheets/users.scss

```diff
@@ -43,6 +43,11 @@
   }
 }
 
+.reducegap {
+  font-weight: bold;
+  margin-bottom:-20px;
+}
+
```

Comment:
That (as well as its usage) seems like an artifact of updating the branch - didn't @ArtOfCode- already review this and asked not to use the override?